style.css

```diff
@@ -494,6 +613,15 @@ nav ul li img {
 
 /* medium screen resolution */
 @media (min-width: 767px) and (max-width: 1000px) {
+  .grid-container {
```

Comment:
Your desktop design is excellent. However, there are discrepancies in its mobile design which require attention. Please refer to the attached video:

Solarized dark             |  Solarized Ocean
:-------------------------:|:-------------------------:
![mobile](https://user-images.githubusercontent.com/69011963/209352888-6889a365-7820-42f8-bc70-24f9aa74cd69.gif)   |  ![Template 1-Mobile-Detail](https://user-images.githubusercontent.com/69011963/209354108-05d05523-4265-4c33-bb30-6409f76c3e67.png)

- [ ] 1. If I hower your mobile (modal menu) it breaking the design. 
- [ ] 2. Your text is a bit longer than the original design. I suggest you to limit it or you can use overflow method. 
- [ ] 3. Your stacks (html, css, javaScript) are alse really close to the description.
- [ ] 4. As you can see in the video your buttons are also looks different.

Kindly fix all the mentioned changes 😉